deployment/terraform/iam.tf

```
@@ -15,10 +18,30 @@ data "aws_iam_policy_document" "ecs_task_execution_assume_role" {
 
 resource "aws_iam_role" "ecs_task_execution_role" {
   name               = "ecs${var.environment}TaskExecutionRole"
-  assume_role_policy = "${data.aws_iam_policy_document.ecs_task_execution_assume_role.json}"
+  assume_role_policy = "${data.aws_iam_policy_document.ecs_assume_role.json}"
 }
 
 resource "aws_iam_role_policy_attachment" "ecs_task_execution_role_policy" {
   role       = "${aws_iam_role.ecs_task_execution_role.name}"
   policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
 }
+
+data "aws_iam_policy_document" "ses_send_email" {
+  statement {
+    effect = "Allow"
+
+    resources = ["*"]
+    actions   = ["ses:SendRawEmail"]
+  }
+}
+
+resource "aws_iam_role" "ecs_task_role" {
+  name               = "ecs${var.environment}TaskRole"
+  assume_role_policy = "${data.aws_iam_policy_document.ecs_assume_role.json}"
+}
+
+resource "aws_iam_role_policy" "ses_send_email_policy" {
```

I would consider dropping the `_policy` suffix from this resource name since it is redundant (the resource is named `aws_iam_role_policy`).

```suggestion
resource "aws_iam_role_policy" "ses_send_email" {
```